.zuul.yaml

```diff
@@ -70,6 +70,8 @@
             disk: 20
             vcpus: 1
       cifmw_test_operator_tempest_ntp_extra_images: https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img
+      cifmw_test_operator_tempest_network_attachments:
+        - ctlplane
```

Comment:
+1 for adding test coverage for this

eventually we need to have envtest coverage of the operator but this will prevent this regressing and prove that the pr actually fixes it so lets see what ci says